AI Game Jam/Assets/Scripts/ItemGoal.cs

```diff
@@ -0,0 +1,39 @@
+/*
+* Description: This script is used to manage the area where the player can place items. It also shows hints for the goals when the player is in range of the goal.
+* Author: Chase Bennett-Hill
+* Last Modified: 13 / 03 / 24
+* Last Modified By: Chase Bennett-Hill
+*/
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class ItemGoal : MonoBehaviour
+{
+    public string itemName; //what item can be placed here
+    
+    public GameObject hint; //the item that can be placed here
+    // Start is called before the first frame update
+    public string hintNoItem = "";
+    public string hintWithItem = "";
+
+    public void UseObject()
+    {
+       if(gameObject.name.Contains("Door"))
```

Comment:
Using comments, tell me what is happening in sequence! Even though each line is a simple and understandable piece of code, since what is happening is sequential, it's good to know where to follow along in case a bug happens somewhere in the middle!

Reply:
Also you started commenting at the start of this script, best to keep it consistent.